Automattic/wp-calypso
# Update default theme for professional sites to pub/professional-business

#### Changes proposed in this Pull Request

Related to Improved Onboarding phase 2. More context available on paObgF-b3-p2.

Let's give professional sites a proper theme.

*Before this diff:*

<img width="770" alt="Zrzut ekranu 2019-05-29 o 18 14 19" src="https://user-images.githubusercontent.com/205419/58573226-98390380-823d-11e9-80d4-6748b28ba022.png">

*After this diff:*

<img width="988" alt="Zrzut ekranu 2019-05-29 o 18 36 34" src="https://user-images.githubusercontent.com/205419/58574651-beac6e00-8240-11e9-9dd3-2375a0fd089a.png">

#### Testing instructions

1. Choose "Professional" in the segmentation step.
2. Go through the entire signup flow
3. Confirm the final site looks like on the "After" screenshot above
